apps/web/src/pages/get-started/components/OpenWorkflowButton.tsx

```diff
@@ -1,14 +1,28 @@
+import { errorMessage } from '@novu/design-system';
 import { INotificationTemplate } from '@novu/shared';
 
-import { ROUTES } from '../../../constants/routes.enum';
-import { useCreateTemplateFromBlueprint } from '../../../api/hooks';
-import { errorMessage } from '../../../utils/notifications';
-import { getBlueprintTemplateById, getTemplateById } from '../../../api/notification-templates';
-import { getWorkflowBlueprintDetails } from '../../../utils';
 import { TemplateCreationSourceEnum } from '../../templates/shared';
 import { useSegment } from '../../../components/providers/SegmentProvider';
 import { OnboardingWorkflowRouteEnum } from '../consts/types';
 import { StyledLink } from '../consts/shared';
+import { useCreateWorkflowFromBlueprint } from '../../../hooks';
+import { openInNewTab } from '../../../utils';
+import { parseUrl } from '../../../utils/routeUtils';
+import { ROUTES } from '../../../constants/routes.enum';
+
+function buildWorkflowEditorUrl(workflow: INotificationTemplate, step?: OnboardingWorkflowRouteEnum) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** Consider moving to its own file as well‏